README.md

```diff
@@ -102,9 +102,9 @@ Image is set as `docker.io/aveshasystems/worker-operator:$(VERSION)` in the Make
    docker exec -it kind-control-plane crictl images
    ```
 
-### Deploy on a Cluster
+### Deploy the Worker Operator on a Cluster
 
-Create a chart values file, `yourvaluesfile.yaml`.
+Create the chart values file called `yourvaluesfile.yaml`.
```

Comment:
When you're creating, you can use the indefinite article as in create a slice, create a folder.. 

so `Create a chart values file ..` holds good here.